### ISSUE TITLE
Ember trying to assign ID twice

### ISSUE THREAD

**Issue by kpie89** [ga-wdi-boston/capstone-project#125]
I am passing an object up the tree to my song-search route which then does a post for /songs. However, it gets passed a property called "id" which I want to be assigned to a property called "soundcloud_id" in my back end. However, when ember does the post it assigns what it gets as id to the "song_id" rather than the "soundcloud_id" and then tries to assign the "song_id" but fails because it already has an ID assigned. I understand this is sort of hard to follow. In essence, I suppose I have to explicitly tell ember what "soundcloud_id" is before passing it in as params for my post request?

**Comment by jrhorn424** [comment]
Any updates? Issue text seems gone.

**Comment by kpie89** [comment]
The problem was that I was passing an entire Ember object of "song" from the data I was recieving from the soundcloud API. Instead, I simply parsed out the data in the song search route and told ember explicitly what the params equaled.

**Comment by kpie89** [comment]
Eessentially.
